### ISSUE TITLE
Most effective parameter for finetuning

### ISSUE THREAD

**Issue by Rishav-hub** [FlagOpen/FlagEmbedding#151]
Hello, is there any table that should the effective parameter configuration while training for better performance? 

I am talking about these parameter -:

```
--learning_rate 1e-5 \
--fp16 \
--num_train_epochs 5 \
--per_device_train_batch_size 8 \
--per_device_eval_batch_size 8 \
--dataloader_drop_last True \
--normlized True \
--temperature 0.02 \
--query_max_len 64 \
--passage_max_len 512 \
--train_group_size 2 \
--negatives_cross_device \
--logging_steps 10 \
```

**Comment by Rishav-hub** [comment]
Also, I have observed that keeping the per_device_train_batch_size 1, gives the final train_loss less compared to keeping it per_device_train_batch_size as 8

**Comment by staoxiao** [comment]
For some suggestions, you can refer to https://github.com/FlagOpen/FlagEmbedding/tree/master/examples/finetune#3-train. The most important arguments are `per_device_train_batch_size`, `train_group_size` .
We use in-batch negatives strategy, where a larger `per_device_train_batch_size` will bring more negatives to compute loss, so the training loss will be higher when increasing the batch size.  But with more negatives for training,  the performance will be better. So we suggest using a large batch size.
For `train_group_size`, we suggest to mind hard negatives and set it to 2 or 8.

**Comment by Rishav-hub** [comment]
Thank you it clears my doubt!!
